lib/hashtable.cc

```diff
@@ -122,6 +122,63 @@ consume_fasta(
     delete parser;
 }
 
+void Hashtable::consume_fasta_bitsplit(
```

Comment:
bitsplit doesn't mean anything to me :-/ Coudl we call it `consume_fasta_sampling` or some such?